static/skin/taskbar.js

```diff
@@ -7,7 +7,10 @@ const jq = jQuery.noConflict(true);
 jq(document).ready(() => {
     (function ($) {
         const root = $( `link[type='root']` ).attr("href");
-        const bookName = window.location.pathname.split(`${root}/`)[1].split('/')[0];
+
+        const bookName = (window.location.pathname.search(`${root}/search`) != -1) ? 
```

Comment:
I guess one of my previous comments was not straightforward enough. Don't `search` in the pathname, because that may match URLs that are different from `${root}/search`. Simply

```JS
const bookName = (window.location.pathname == `${root}/search`)
               ? (new URLSearchParams(window.location.search)).get('content') 
               : window.location.pathname.split(`${root}/`)[1].split('/')[0];
```